frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -252,10 +248,10 @@ const PropertyListView: React.FC = () => {
   const dispatch = useDispatch();
 
   const fetch = (accept: 'csv' | 'excel') => {
-    const query = getServerQuery(true, { pageIndex, pageSize, filter, agencyIds });
+    const query = getServerQuery({ pageIndex, pageSize, filter, agencyIds });
     return dispatch(
       download({
-        url: getPropertyReportUrl({ ...query, all: true }),
+        url: getPropertyReportUrl({ ...query, all: true, propertyType: undefined }),
```

Comment:
So this puts both property types into the excel export?

Reply:
Yup, same idea as the previous `ignorePropertyType` but this one only affect the excel/csv reports allowing the normal list view to default to land again

Reply:
perfect 👍